src/controllers/processors/SessionCleanupProcessor.ts

```diff
@@ -0,0 +1,38 @@
+import { Session } from 'ch-node-session-handler';
+import { Request } from 'express';
+import { provide } from 'inversify-binding-decorators';
+
+import { FormActionProcessor } from 'app/controllers/processors/FormActionProcessor';
+import { ApplicationData, APPLICATION_DATA_KEY } from 'app/models/ApplicationData';
+import { APPLICATION_DATA_UNDEFINED, SESSION_NOT_FOUND_ERROR } from 'app/utils/CommonErrors';
+import { CONFIRMATION_PAGE_URI } from 'app/utils/Paths';
+
+@provide(SessionCleanupProcessor)
```

Comment:
Maybe add one info/debug logging to inform that session has been cleaned up. What do you think?